fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java

```diff
@@ -354,6 +369,14 @@ public static Metrics buildDataFileMetrics(TIcebergDataFile dataFile) {
             }
         }
 
+        for (Types.NestedField field : nativeTable.schema().columns()) {
+            if (field.type() instanceof Types.DecimalType || field.type() == Types.UUIDType.get()) {
+                //change to BigEndian
+                reverseBuffer(lowerBounds.get(field.fieldId()));
```

Comment:
you can use this to change to BigEndian **`buffer.order(ByteOrder.BIG_ENDIAN);`**